### DISCUSSION TITLE
Ariakit examples

### DISCUSSION THREAD

**Discussion by gianlazaro** [ariakit/ariakit#3259] [General]
I'm trying to figure out if this should be an issue or not but I thought someone should be aware that most examples on StackBlitz that use Next.JS Parallel Routes are broken (https://ariakit.org/examples/dialog-next-router) (https://ariakit.org/examples/tab-next-router)

**Comment by diegohaz** [comment]
Thanks for reporting. It should work again in a few minutes.
